packages/infra/lib/api-stack.ts

```diff
@@ -1114,7 +1114,7 @@ export class APIStack extends Stack {
         ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
       },
       layers: [lambdaLayers.shared, lambdaLayers.chromium],
-      memory: 1024,
+      memory: 4096,
```

Comment:
This too, from [this merge commit](https://github.com/metriport/metriport/pull/1187/commits/1c6c5c412bc411838a44ab0f3610627ac97f6902)